app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -464,7 +464,7 @@ def parse_networks(args)
         # Convert network id into name
         net = dc_networks.detect { |n| [n.id, n.name].include?(interface['network']) }
```

Comment:
Should this also be changed to first try all IDs and otherwise fall back to a name? I'm wondering if there's a case where the first network has the ID of a later network and you could end up with a mismatch.

Reply:
the network id should be always unique.
It could happen though in rare cases that you name your network in a manner of the vmware network id (i.e. `network_4302`) and it could end up selected. I left this for `hammer` as now if you have in some hammer commands a network name, it works, but it wouldn't now. But we can iterate twice. It will probably be better, to avoid the weird naming usecases...